integration/setup/setup.go

```diff
@@ -61,10 +62,43 @@ func installResources(ctx context.Context, config Config) error {
 	}
 
 	{
-		err = config.Release.InstallOperator(ctx, key.AppOperatorReleaseName(), release.NewVersion(env.CircleSHA()), templates.AppOperatorValues, v1alpha1.NewAppCRD())
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("installing app-operator"))
+
+		operatorVersion := fmt.Sprintf("1.0.0-%s", env.CircleSHA())
```

Comment:
Are you using 1.0.0 specifically on purpose?

Reply:
Yes this is because its deployed by draughtsman. I added a TODO comment and this will be cleaned up when the operator is flattened.